src/main.py

```diff
@@ -453,7 +446,7 @@ def postprocess(data):
 VOR = copy.deepcopy(VOR_SNIPPET)
 VOR.update(OrderedDict([
```

Comment:
Nope ;) I updated the jira ticket with some of the facts that came out of the email conversation, one of which is that POAs don't have related articles.